### PR TITLE
Added more clarifications

### PR DESCRIPTION
**Pull request by hannanelson**

### CODE REVIEW

**Review by melamedjoel — DISMISSED** [review]
LGTM

**Review by schinthakindims — DISMISSED** [review]
LGTM

File: modules/ROOT/pages/mq-queues.adoc [hunk 1]
```diff
@@ -344,9 +346,23 @@ To preserve message order for an app, make sure that each app uses a specific me
 If a message in a message group is not acknowledged, it returns to the queue 
 and Anypoint MQ continues processing the next message in the group.
 This behavior might not be desirable if a message in the group depends on a previous message.
-To ensure that the messages are processed in order, set the value of `maxConcurrency` on the flow to `1`, which forces the flow to process requests one at a time.
 
-If a message doesn't have an associated message group ID, the message is assigned to a default message group. If you require a single group of ordered messages, provide the same message group ID for messages sent to the FIFO queue.
+To ensure that the FIFO queue processes the messages in order:
+
+. Set the value of `maxConcurrency` on the flow to `1`, which forces the flow to process requests one at a time.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/mq-queues.adoc]: process requests individually? I am not sure if "one at a time" translates properly.

**Review by sarathecoubian — DISMISSED** [review]
Looks good. Optional comment.

File: modules/ROOT/pages/mq-queues.adoc (hunk 1 above)
Comment (hannanelson) [reply modules/ROOT/pages/mq-queues.adoc]: Good point. Kim never complained about that, but one word is almost always better than 4!

**Review by sarathecoubian — APPROVED** [review]

**Pull request merged**